src/state/chain/atoms/chainAtoms.ts

```diff
@@ -81,8 +81,18 @@ export const GRAPH_CLIENTS = {
     import.meta.env.VITE_SUBGRAPH_URL || SUBGRAPH_URL[ChainId.Mainnet]
   ),
   [ChainId.Base]: new GraphQLClient(SUBGRAPH_URL[ChainId.Base]),
+  CONTENTFUL: new GraphQLClient(
+    `https://graphql.contentful.com/content/v1/spaces/9pqtywannd90/environments/master`,
+    {
+      headers: {
+        authorization: `Bearer xyXOpKn3d21kQfTNCmxb0eGTNU1eLnEVkGuPRCDkiQI`,
```

Comment:
Maybe it would be good to move this to an env variable, in case it expires and we have to change it, so we don't have to change the code